src/App.tsx

```diff
@@ -1,25 +1,45 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
```

Comment:
The `items` array is defined outside of the App component, which means it's created once and never changes. If you plan to make this array dynamic (for example, fetching data from an API), it would be better to move it inside the App component and use the `useState` or `useEffect` hooks to handle data fetching and state updates.